vitrina/cms/cms_plugins.py

```diff
@@ -0,0 +1,124 @@
+        if instance.page.node.get_children():
+            parent = instance.page.node
+            children = instance.page.node.get_children()
+        else:
+            parent = instance.page.node.get_parent()
+            children = instance.page.node.get_siblings()
+        context.update({
+            'children': children,
+            'parent': parent
+        })
+        return context
+
+
+@plugin_pool.register_plugin
+class LearningMaterialPlugin(CMSPluginBase):
+    model = CMSPlugin
+    module = _("ADP")
+    name = _("Mokymosi medžiagos")
+    render_template = "pages/learning_material.html"
+
+    def render(self, context, instance, placeholder):
+        context = super().render(context, instance, placeholder)
+        context.update({
+            'items': LearningMaterial.objects.filter(
+                published__isnull=False
+            ).order_by('-published'),
+        })
+        return context
+
+
+@plugin_pool.register_plugin
+class ReportPlugin(CMSPluginBase):
+    model = CMSPlugin
+    module = _("ADP")
+    name = _("Ataskaitos")
+    render_template = "pages/report.html"
+
+    def render(self, context, instance, placeholder):
+        context = super().render(context, instance, placeholder)
+        context.update({
+            'items': PublishedReport.objects.all()
+        })
+        return context
+
+
+@plugin_pool.register_plugin
+class EUCommissionPortalPlugin(CMSPluginBase):
+    model = CMSPlugin
+    module = _("ADP")
+    name = _("Europos komisijos portalai")
+    render_template = "pages/other.html"
+
+    def render(self, context, instance, placeholder):
+        context = super().render(context, instance, placeholder)
+        context.update({
+            'title': _("Europos komisijos portalai"),
+            'items': ExternalSite.objects.filter(type=ExternalSite.EU_COMISSION_PORTAL),
+        })
+        return context
+
+
+@plugin_pool.register_plugin
+class EULandPlugin(CMSPluginBase):
+    model = CMSPlugin
+    module = _("ADP")
+    name = _("EU šalys")
+    render_template = "pages/other.html"
+
+    def render(self, context, instance, placeholder):
+        context = super().render(context, instance, placeholder)
+        context.update({
+            'title': _("EU šalys"),
+            'items': ExternalSite.objects.filter(type=ExternalSite.EU_LAND),
+        })
+        return context
+
+
+@plugin_pool.register_plugin
+class OtherLandPlugin(CMSPluginBase):
+    model = CMSPlugin
+    module = _("ADP")
+    name = _("Kitos šalys")
+    render_template = "pages/other.html"
+
+    def render(self, context, instance, placeholder):
+        context = super().render(context, instance, placeholder)
+        context.update({
+            'title': _("Kitos šalys"),
+            'items': ExternalSite.objects.filter(type=ExternalSite.OTHER_LAND),
+        })
+        return context
+
+
+@plugin_pool.register_plugin
+class FaqPlugin(CMSPluginBase):
```

Comment:
I believe it shows id of plugin object. 

http://127.0.0.1:8000/opening-tips/opening/learningmaterial/
http://127.0.0.1:8000/opening-tips/opening_faq/
http://127.0.0.1:8000/daugiau/reports/
http://127.0.0.1:8000/daugiau/other/

All these pages have custom plugins, which on render collect data, for example all Faq objects, and display them in a page, so that if we add another Faq object in admin, it would automatically be shown in this page. However if for example Faq objects won't be managed from admin, then I can change it to text plugin.

Reply:
Is it possible to make content editable both, from admin and inline, by double clicking on it? One of the main Django CMS feature, is that you can edit pages by double clicking on them, it would be nice to have that.